src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py

```diff
@@ -7,15 +7,288 @@
 from neuroconv.utils.dict import DeepDict
 
 
-class BrezovecMultiPlaneImagingInterface(BaseImagingExtractorInterface):
+class BrezovecFunctionalGreenImagingInterface(BaseImagingExtractorInterface):
     """
-    Data Interface for writing the Clandinin lab imaging data to NWB file using BrezovecMultiPlaneImagingExtractor.
+    Data Interface for writing the functional imaging data (GCaMP6f) for Clandinin lab to NWB file using BrezovecMultiPlaneImagingExtractor.
     """
 
     Extractor = BrezovecMultiPlaneImagingExtractor
 
     def __init__(self, folder_path: FolderPathType, stream_name: str, verbose: bool = True):
+        """
+        Initialize reading of NIfTI files.
+
+        Parameters
+        ----------
+        folder_path : FolderPathType
+            The path to the folder that contains the NIfTI image files (.nii) and configuration files (.xml) from Bruker system.
+        stream_name: str, optional
+            The name of the recording channel.
+        verbose : bool, default: True
+        """
+        super().__init__(
+            folder_path=folder_path,
+            stream_name=stream_name,
+            verbose=verbose,
+        )
+
+    @classmethod
+    def get_streams(cls, folder_path) -> dict:
+        streams = BrezovecMultiPlaneImagingExtractor.get_streams(folder_path=folder_path)
+        return streams
+
+    def get_metadata(self) -> DeepDict:
+        metadata = super().get_metadata()
+
+        xml_metadata = self.imaging_extractor.xml_metadata
+        session_start_time = parse(xml_metadata["date"])
+        metadata["NWBFile"].update(session_start_time=session_start_time)
+
+        description = f"Version {xml_metadata['version']}"
+        device_name = "BrukerFluorescenceMicroscope"  # TODO doucle check in the paper
+        metadata["Ophys"]["Device"][0].update(
+            name=device_name,
+            description=description,
+        )
+
+        imaging_plane_metadata = metadata["Ophys"]["ImagingPlane"][0]
+        imaging_plane_metadata.update(
+            # optical_channel =
+            #     dict(
+            #         name=self.imaging_extractor.stream_name,
+            #         emission_lambda=525, #513 FPbase
+            #         description="Green channel of the microscope, 525/50 nm filter.",
+            #     ),
+            device=device_name,
+            imaging_rate=self.imaging_extractor.get_sampling_frequency(),
+            excitation_lambda=920.0,
+            indicator="GCaMP6f",
+        )
+        two_photon_series_metadata = metadata["Ophys"]["TwoPhotonSeries"][0]
+        two_photon_series_metadata.update(
+            name="FunctionalGreenTwoPhotonSeries",
```

Comment:
```suggestion
            name="TwoPhotonSeriesFunctionalGreen",
```

This is inverted with respect to best practices which is the other way around. Check this:
https://nwbinspector.readthedocs.io/en/dev/best_practices/general.html

Reply:
Once you correct or address some points you should comment here that you have done it so we can keep track of this.

Reply:
yes sorry!